frontend/appflowy_flutter/lib/plugins/document/presentation/plugins/code_block/code_block_shortcut_event.dart

```diff
@@ -123,3 +131,27 @@ SelectionMenuItem codeBlockMenuItem = SelectionMenuItem(
     editorState.apply(transaction);
   },
 );
+
+ShortcutEventHandler _selectAllHandler = (editorState, event) {
+  final nodes = editorState.service.selectionService.currentSelectedNodes;
+  final codeBlockNodes =
+      nodes.whereType<TextNode>().where((node) => node.id == kCodeBlockType);
+  if (codeBlockNodes.isEmpty || codeBlockNodes.length != 1) {
+    return KeyEventResult.ignored;
+  }
+
+  final codeBlockNode = codeBlockNodes.first;
+  editorState.service.selectionService.updateSelection(
+    Selection(
+      start: Position(
+        path: codeBlockNode.path,
+      ),
+      end: Position(
+        path: codeBlockNode.path,
+        offset: codeBlockNode.toPlainText().length,
```

Comment:
We might want to add a `.length` property to `TextNode`. Calling `toPlainText().length` looks pretty common 

cc @LucasXu0